specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-10-02-preview/managedClusters.json

```diff
@@ -1434,16 +1447,29 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "NoContent"
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ManagedCluster"
+            }
           },
           "default": {
-            "description": "Error response describing why the operation failed. If the cluster doesn't exist, 404 (Not found) is returned.",
```

Comment:
Cluster not existing is still a 404, correct?

Reply:
Yes, just thought it was unnecessary to be explicitly included here in the description. Happy to hear a different opinion if you think we should keep that.

Reply:
No, it's fine to remove this document. I just wanted to confirm it wasn't because of a behavior change.